lib/target_mvd/targets.ex

```diff
@@ -47,9 +47,16 @@ defmodule TargetMvd.Targets do
   end
 
   def create_target(attrs \\ %{}) do
-    %Target{}
-    |> Target.changeset(attrs)
-    |> Repo.insert()
+    user = User |> Repo.get!(attrs["user_id"]) |> Repo.preload(:targets)
+    targets = Enum.count(user.targets)
```

Comment:
How about doing the query directly?
```suggestion
    user_targets = Target |> where(user_id: attrs["user_id"]) |> Repo.aggregate(:count)
```